.github/workflows/updatecli.yaml

```diff
@@ -22,19 +20,23 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-2
+      - name: Install updatecli CLI
```

Comment:
probably covered somewhere else but there should be an installer action that just installs the binary doesn't try run it in a container, `setup-updatecli` that way dependabot can update it

Reply:
Yup, I should have some free time this week end for that \o/